app/src/pages/inside/logsPage/historyLine/historyLineItem/historyLineItem.jsx

```diff
@@ -30,21 +30,22 @@ import styles from './historyLineItem.scss';
 
 const cx = classNames.bind(styles);
 
-const HistoryLineItemContent = ({
+export const HistoryLineItemContent = ({
   active,
   status,
   statistics,
   launchNumber,
   hasChildren,
   growthDuration,
+  showTriangles,
   ...rest
 }) => {
   const isNotEmpty = !hasChildren ? Object.keys(statistics.defects).length : false;
 
   return (
     <div className={cx('history-line-item-content')}>
       <div className={cx('triangles-wrapper')}>
-        <Triangles growthDuration={growthDuration} />
+        <Triangles growthDuration={growthDuration} showTriangles />
```

Comment:
How it works now? Does the `Triangles` component has the `showTriangles` prop?
I suggest not displaying this component at all if `showTriangles` is false:
```
{showTriangles && (
  <div className={cx('triangles-wrapper')}>
     <Triangles growthDuration={growthDuration} />
  </div>
)}
```